src/variables/investments.jl

```diff
@@ -52,26 +52,19 @@ function add_investment_variables!(model, graph, sets, variables)
             row -> row.investment_integer,
         ),
         (
-            :assets_decommission_simple_method,
-            row -> (row.milestone_year, row.asset),
+            :assets_decommission,
+            row -> (row.asset, row.milestone_year, row.commission_year),
             _ -> 0.0,
             _ -> Inf,
             row -> row.investment_integer,
         ),
         (
-            :flows_decommission_using_simple_method,
-            row -> (row.milestone_year, (row.from_asset, row.to_asset)),
+            :flows_decommission,
+            row -> ((row.from_asset, row.to_asset), row.milestone_year, row.commission_year),
             _ -> 0.0,
             _ -> Inf,
             _ -> false,
```

Comment:
```suggestion
            _ -> row.investment_integer,
```

`flows_decommission` should also depend on the `investment_integer`, same as the investment.

Reply:
Fixed locally, please check again after update.